tools/fiptool/fiptool.c

```diff
@@ -753,6 +773,7 @@ static void create_usage(void)
 	printf("\n");
 	printf("Options:\n");
 	printf("  --align <value>\t\tEach image is aligned to <value> (default: 1).\n");
+	printf("  --align-reserve <value>\t\tEach image is padded by <value> (default: 0).\n");
```

Comment:
As you wish.
Can you think of any ways to improve the wording while at it? I clearly copied the --align example, but it might be clearer to append "bytes" to both? I.e., to distinguish it from which byte value is used to pad up to the alignment size, which I assume is always 0x00.